frontends/p4/typeChecking/typeUnification.cpp

```diff
@@ -123,9 +123,9 @@ bool TypeUnification::unifyCall(const IR::Node* errorPosition,
             ++paramIt;
     }
 
-    // Check remaining parameters: they must be all optional
+    // Check remaining parameters: they must be all optional or have default values
     for (auto p : left) {
-        bool opt = p.second->isOptional();
+        bool opt = p.second->isOptional() || p.second->defaultValue != nullptr;
```

Comment:
+1. And soon we can get rid of @optional!

Reply:
At this point we don't have anything to replace optional.